ts/nni_manager/rest_server/restHandler.ts

```diff
@@ -317,6 +318,15 @@ class NNIRestHandler {
         });
     }
 
+    private stop(router: Router): void {
+        router.delete('/experiment', (req: Request, res: Response) => {
+            this.nniManager.stopExperimentTopHalf().then(() => {
+                res.send();
+                this.nniManager.stopExperimentBottomHalf();
```

Comment:
Maybe we can wait for the whole experiment to stop, then `res.send()`? To avoid something like the dispatcher has an independent process in the future. Or we need to put this kind of independent process `cleanup()` in `stopExperimentTopHalf()`? Anyway, it is fine in this version.

Reply:
Because killing trials is too time consuming. I'm afraid users (like Quanlu) will kill the process with second ctrl-c if we let them wait that long.